### DISCUSSION TITLE
Ft4 ? Any plan ?

### DISCUSSION THREAD

**Discussion by jreytinat** [N0BOY/FT8CN#128] [Ideas]
Hello
I love this app!
It saved me several sota on 10m.
Any plan to code a ft4cn?
Thanks a million!!!
Gr8 job

**Comment by jreytinat** [comment]
Aka F4ivi

**Reply by HB9HCI** [reply]
FT4: same here! 
And it would be also very handy if the field for the API-Call for QRZ would be a Combo-Box: I have a lot of logbooks in QRZ (DL/HB9HCI HB9HCI/P F/HB9HCI ...) each has its own API key.

**Comment by danleetw** [comment]
Please try this one
https://github.com/danleetw/FT8TW
